src/OSPSuite.UI/Services/DataImporter.cs

```diff
@@ -47,10 +47,13 @@ public IList<MetaDataCategory> DefaultMetaDataCategories()
 
          var organCategory = createMetaDataCategory<string>(Constants.ObservedData.ORGAN, isMandatory: true, isListOfValuesFixed: true);
          organCategory.Description = ObservedData.ObservedDataOrganDescription;
+         organCategory.TopNames.Add(Constants.ObservedData.PERIPHERAL_VENOUS_BLOOD_ORGAN);
```

Comment:
I don't quite understand why this is being done in CORE and not in the caller. What happens if this is also done in PKSim? Will the entry be duplicated? If feels to me that the order should be dictated by the caller of and not by the  core... @Yuri05 thoughts?

Reply:
Yep, think you are right. If the caller fills the initial list it's better to add "top names" by the caller as well

Reply:
@Yuri05 and @msevestre I am not sure. On the one hand I agree that the caller should be managing this but on the second hand, keep in mind that DefaultMetaDataCategories is only giving the callers a default (or base if you would prefer) configuration for the MetaDataCategories and the caller could then modify anything such as the "top names". Of course if you think in MoBi these "top names" do not make sense I would say we should move them to the PKSim side only but if it does make sense, and I think it does right?, then we should keep them here since the callers could still further edit these MetaDataCategories before feeding them back to Core.